docs/buttons.md

```diff
@@ -36,11 +36,11 @@ _Mixin which helps to create button with specified border and background color w
 - `$active-border` - defines button's border color when active (**default `darken($border, 12.5%)`**)
 
 ### Usage: 
-Assign primary color (variable) as button's text and background color and primary dark (variable) as border of button 
+Assign primary `$color-red` as background color and `$color-red-dark` color `$color-red-dark` as border of button 
```

Comment:
Assign `$color-red` as background color and `$color-red-dark` as border color of button.